docs/docs-beta/docs/getting-started/installation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Installing Dagster"
+description: "Learn how to install Dagster"
+sidebar_position: 20
+sidebar_label: "Installation"
+---
+
+# Installing Dagster
+
+To follow the steps in this guide, you'll need:
+
+- To install Python 3.8 or higher. **Python 3.11 is recommended**.
+- To install pip, a Python package installer
+
+## Setting up a virtual environment
+
+After installing Python, it's recommended that you set up a virtual environment. This will isolate your Dagster project from the rest of your system and make it easier to manage dependencies.
+
+There are many ways to do this, but this guide will use `venv` as it doesn't require additional dependencies.
+
+```bash
+python -m venv .venv
+source .venv/bin/activate
+```
+
+:::tip
+**Looking for something more powerful than `venv`?** Try `pyenv` or `pyenv-virtualenv`, which can help you manage multiple versions of Python on a single machine. Learn more in the [pyenv GitHub repository](https://github.com/pyenv/pyenv).
+:::
+
+## Installing Dagster
+
+To install Dagster in your virtual environment, open your terminal and run the following command:
+
+```bash
+pip install dagster dagster-webserver
+```
+
+This command will install the core Dagster library and the webserver, which is used to serve the Dagster UI.
+
+## Verifying installation
+
+To verify that Dagster is installed correctly, run the following command:
+
+```bash
+dagster --version
+```
+
+The version numbers of Dagster should be printed in the terminal:
+
+```bash
+> dagster --version
+dagster, version 1.8.4
+```
+
+If you see `dagster, version 1!0+dev`, then you have the development version of Dagster installed. You can install the latest stable version by providing a version to the `pip install` command, for example:
+
+```bash
+pip install 'dagster==1.8.4'
+```
```

Comment:
hm, did we call this out in the past? I think it's unlikely that end users will have `pip install -e .[dev]` the dagster repo, but I may be mistaken.

Reply:
I don't think so; this was in here when I started editing. I don't think I've ever seen it come up either so I'm fine to take it out.